tensorboard/plugins/mesh/demo_utils.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utils used for the mesh demo."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+
+def _parse_vertex(vertex_row):
+  """Parses a line in a PLY file which encodes a vertex coordinates.
+
+  Args:
+    vertex_row: string with vertex coordinates and color.
+
+  Returns:
+    `[3,]` array of vertex coordinates and `[3,]` array with RGB color.
```

Comment:
Here you use “array” to mean `list`, but below you use it to instead
mean `ndarray`. Suggest rephrasing to:

```
  Returns:
    2-tuple containing a length-3 array of vertex coordinates (as
    floats) and a length-3 array of RGB color values (as ints between 0
    and 255, inclusive).
```

Speaking of which, shouldn’t the colors be returned as `int`s?